### DISCUSSION TITLE
Make image continue onto next page ignoring page break

### DISCUSSION THREAD

**Discussion by bliitzkrieg** [dolanmiu/docx#1878] [Q&A] [answered]
Is there a way to make an image continue onto a new page ignoring the page break? Currently it's cutting vertically long images off.

![image](https://user-images.githubusercontent.com/2092432/212201505-e900ed66-4112-4f83-a3f1-772213464019.png)

**Comment by dolanmiu** [comment, marked as answer]
Do you mean adding a Footer? Adding headers and footers are possible, it may solve your issue
